sdcm/tester.py

```diff
@@ -796,6 +796,8 @@ def is_encrypt_keys_needed(self):
         )
 
     def prepare_kms_host(self) -> None:
+        if self.params.is_enterprise and self.params.get('cluster_backend') == 'aws' and not self.params.get('scylla_encryption_options'):
+            self.params['scylla_encryption_options'] = "{ 'cipher_algorithm' : 'AES/ECB/PKCS5Padding', 'secret_key_strength' : 128, 'key_provider': 'KmsKeyProviderFactory', 'kms_host': 'auto'}"  # pylint: disable=line-too-long
```

Comment:
Probably would be better to read these from a configuration file.
But can be done in a followup PR if we agree on that.